src/systemtest/resources/features/SourceChangeFlow.feature

```diff
@@ -10,9 +10,36 @@ Feature: Source Change system test
     And a jenkins job '"ActT1Job"' from '"src/systemtest/resources/JenkinsShellScripts/ActT1Script.txt"' is created with parameters: EVENT_ID,
     And a jenkins job '"ActSFCJob"' from '"src/systemtest/resources/JenkinsShellScripts/ActSFCScript.txt"' is created with parameters: EVENT_ID,
     And a jenkins job '"CLMJob"' from '"src/systemtest/resources/JenkinsShellScripts/CLMScript.txt"' is created with parameters: EVENT_ID,
+    And a jenkins job '"FlowCompleteJob"' from '"src/systemtest/resources/JenkinsShellScripts/FlowCompleteScript.txt"' is created with parameters: ,
 
-    ### Add Subscription to EI ###
+    ### Add CLMSubscription to EI ###
+    Given subscription object "CLMSubscription" is created which will trigger "CLMJob" with parameters
+    When notification with key "EVENT_ID" and value "id" is added to "CLMSubscription"
+    And condition with jmespath "submission.gitIdentifier.repoName=='my-repo'" is added to "CLMSubscription"
+    And condition with jmespath "submission.submitter.email=='jane@example.com'" is added to "CLMSubscription"
+    Then we send the "CLMSubscription" to eiffel intelligence for creation.
 
+    ### Add ActT1Subscription to EI ###
+    Given subscription object "ActT1Subscription" is created which will trigger "ActT1Job" with parameters
+    When notification with key "EVENT_ID" and value "submission.confidenceLevels[0].eventId" is added to "ActT1Subscription"
```

Comment:
confidence levels have also names so create rule that extracts eventId for confidence level with given name.

Reply:
Done

Reply:
I thought a job had already been added to the Travis build which runs the system tests, but apparently not. I'll create a separate issue for this.